Ochowo/iReporter
# #162179956 create api endpoints

**What does this PR do?**
- Creates an  endpoint to 
   - get all  redflags
   - get a specific redflag
   - post a redflag
   - patch a redflag
   - delete a redflag

**Description of the task  to be completed**
- Ensure the endpoints below is working as it should
    `GET/api/red-flags/:id`
    `GET/api/red-flags/`
    `POST/api/red-flags/`
    `PATCH/api/red-flags/:id`
    `DELETE/api/red-flags/:id`

**How should this be tested manually**
   After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have 
   nodejs and express installed. Afterwards, run `npm start` to initiate the app.
- test the endpoint using postman with `localhost:3000` as the host

**What are the relevant pivotal stories**
- N/A

**Any background context you want to add?**
- N/A

**Screenshots**
N/A
